.jenkins/Jenkinsfile

```diff
@@ -137,7 +137,8 @@ def win2016CrossCompile(String build_type) {
 }
 
 parallel "Check Developer Experience Ubuntu 16.04" :            { checkDevFlows('16.04') },
-         "Check Developer Experience Ubuntu 18.04" :            { checkDevFlows('18.04') },
+         "Check Developer Experience Ubuntu 18.04" :            { checkDevFlows('18.04') }
+/*
```

Comment:
Was this just for testing?

Reply:
Yes this is for testing , I added it with the commit that enables --warn-unused-vars warnings as error , so that we can run only the failing tests to produce logs that are easier to follow

Reply:
I will open another PR with tackling the --warn-unused-vars issue